ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -391,6 +391,40 @@ The description can be reverted manually if needed:
 +vtep_loopback_description: "VTEP_VXLAN_Tunnel_Source"
 ```
 
+#### Underlay point-to-point L3 ethernet interfaces
+
+The interface description for the L3 point-to-point interfaces changes to `P2P_<peer>_<peer_interface>_VRF_<vrf>`. Previously the description was `P2P_LINK_TO_<peer | upper>_<peer_interface>_vrf_<vrf>`. The VRF suffix is only added when using `uplink_type: p2p-vrfs`.
+
+The interfaces using this are routed uplinks and `p2p_links` defined under `l3_edge` or `core_interfaces`.
+
+Setting the description directly on `p2p_links` defined under `l3_edge` or `core_interfaces` still takes precedence.
+
+The default descriptions can be reverted manually if needed:
+
+```diff
++default_underlay_p2p_ethernet_description: "P2P_LINK_TO_{peer!u}_{peer_interface}{vrf?<_vrf_}"
+```
+
+The description fields support templates. See the input variables documentation for available fields.
+
+#### Underlay point-to-point L3 port-channel interfaces
+
+The interface description for the L3 point-to-point port-channels changes to `P2P_<peer>_<peer_interface>`. Previously the default description was `P2P_LINK_TO_<peer>_<peer_interface>`.
+
+The port-channels using this are `p2p_links` defined under `l3_edge` or `core_interfaces`.
+
+Setting the description directly on `p2p_links` defined under `l3_edge` or `core_interfaces` still takes precedence.
+
+In AVD 5.0.0 it is also possible to control the port-channel description separately by setting`description` key under `port_channel`. It will replace the full port-channel interface description.
```

Comment:
```suggestion
In AVD 5.0.0 it is also possible to control the port-channel description separately by setting the `description` key under `port_channel`. It will replace the full port-channel interface description.
```